certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -1,1392 +1,1457 @@
-          },
-          {
-            "name": "body",
-            "in": "body",
-            "required": false,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "roomId": {
-                  "type": "integer",
-                  "description": "Room",
-                  "title": "Room",
-                  "format": "int64",
-                  "x-ms-dynamic-values": {
-                    "operationId": "GetRoomsOperation",
-                    "value-path": "id",
-                    "value-collection": "value",
-                    "value-title": "name",
-                    "parameters": {
-                      "workspaceId": {
-                        "parameter": "workspaceId"
-                      }
-                    }
-                  },
-                  "x-ms-dynamic-list": {
-                    "operationId": "GetRoomsOperation",
-                    "itemsPath": "value",
-                    "itemValuePath": "id",
-                    "itemTitlePath": "name",
-                    "parameters": {
-                      "workspaceId": {
-                        "parameterReference": "workspaceId"
-                      }
-                    }
-                  }
-                },
-                "title": {
-                  "type": "string",
-                  "description": "Title",
-                  "title": "Title"
-                }
-              },
-              "required": [
-                "roomId"
-              ]
-            }
-          }
-        ]
-      }
-    }
-  },
-  "definitions": {
-    "TemplatesResponse": {
-      "type": "object",
-      "properties": {
-        "next": {
-          "type": "string",
-          "description": "The token used to retrieve the next page of templates"
-        },
-        "value": {
-          "type": "array",
-          "description": "The array of templates",
-          "items": {
-            "type": "object",
-            "properties": {
-              "id": {
-                "type": "string"
-              },
-              "name": {
-                "type": "string"
-              }
-            }
-          }
-        }
-      }
-    }
-  },
-  "parameters": {},
-  "responses": {},
-  "securityDefinitions": {
-    "oauth2-auth": {
-      "type": "oauth2",
-      "flow": "accessCode",
-      "authorizationUrl": "https://app.mural.co/api/public/v1/authorization/oauth2/",
-      "tokenUrl": "https://app.mural.co/api/public/v1/authorization/oauth2/token",
-      "scopes": {
-        "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read": "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
-      }
-    }
-  },
-  "security": [
-    {
-      "oauth2-auth": [
-        "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
-      ]
-    }
-  ],
-  "tags": []
+	"swagger": "2.0",
```

Comment:
@rmiller-mural please fix the indentation for the swagger file. all files should be properly formatted.

Reply:
I ran it through our linter. Fixed the indentation issue and a couple of other minor styling tweaks. Now you can at least see the diff